ts/a11y/sre.ts

```diff
@@ -61,5 +63,15 @@ export const sreReady = new Promise<void>((resolve, reject) => {
         }
         checkSRE();
     }).catch((error: Error) => reject(error.message || error));
-});
+  });
+};
 
+
+/**
+ * The functions we need from SRE.
+ */
+export type SRE = {
+  toEnriched(mml: string): Element;
+  setupEngine(options: OptionList): void;
+  engineSetup(): OptionList;
+};
```

Comment:
See my comment above about changing this to `export declare const SRE: {...}` instead.

I can make a PR to this PR that includes the change here and to `sre-node.ts` if that is easier.

Reply:
This is now in `sre_browser.d.ts`.